src/PrestaShopBundle/Form/Admin/Improve/International/Locations/CountryType.php

```diff
@@ -0,0 +1,189 @@
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\DefaultLanguage;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\TypedRegex;
+use PrestaShop\PrestaShop\Core\Form\ConfigurableFormChoiceProviderInterface;
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use PrestaShopBundle\Form\Admin\Type\ShopChoiceTreeType;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraints\Length;
+
+class CountryType extends AbstractType
+{
+    /**
+     * @var TranslatorInterface
+     */
+    protected $translator;
+
+    /**
+     * @var bool
+     */
+    protected $isMultistoreEnabled;
+
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    protected $currencyChoiceProvider;
+
+    /**
+     * @var ConfigurableFormChoiceProviderInterface
+     */
+    protected $zoneChoiceProvider;
+
+    /**
+     * ZoneType constructor.
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        bool $isMultistoreEnabled,
+        FormChoiceProviderInterface $currencyChoiceProvider,
+        ConfigurableFormChoiceProviderInterface $zoneChoiceProvider
+    ) {
+        $this->translator = $translator;
+        $this->isMultistoreEnabled = $isMultistoreEnabled;
+        $this->currencyChoiceProvider = $currencyChoiceProvider;
+        $this->zoneChoiceProvider = $zoneChoiceProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('name', TranslatableType::class, [
+                'label' => $this->translator->trans('Country', [], 'Admin.Global'),
+                'options' => [
+                    'constraints' => [
+                        new Length([
+                            'max' => 64,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters.',
+                                ['%limit%' => 64],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new TypedRegex([
+                            'type' => 'generic_name',
+                        ]),
+                    ],
+                ],
+                'constraints' => [
+                    new DefaultLanguage(),
+                ],
+            ])
+            ->add('iso_code', TextType::class, [
+                'required' => true,
+                'label' => $this->translator->trans('ISO code', [], 'Admin.International.Feature'),
```

Comment:
```suggestion
                'label' => $this->translator->trans('ISO Code', [], 'Admin.International.Feature'),
```

Same here @Julievrz I don't remember what's the expected syntax for this one?

Reply:
No, we don't need the capital c. :-) Let's use the existing string: "ISO code".